app/src/main/java/com/github/onlynotesswent/ui/user/PublicProfile.kt

```diff
@@ -201,6 +206,12 @@ fun TopProfileBar(
             Icon(imageVector = Icons.AutoMirrored.Outlined.ArrowBack, contentDescription = "Back")
           }
         }
+      },
+      actions = {
```

Comment:
right now the log out button shows for all profile screen (including when you are looking at other users). To fix this you can just if the val "includeBackButton" and only display the log out if this val is false. You could also update the name of this val to "isUserProfileScreen".

Reply:
Nice catch